autopush/router/apns2.py

```diff
@@ -11,8 +11,8 @@
 SANDBOX = 'api.development.push.apple.com'
 SERVER = 'api.push.apple.com'
 
-APNS_PRIORITY_IMMEDIATE = 10
-APNS_PRIORITY_LOW = 5
+APNS_PRIORITY_IMMEDIATE = '10'
```

Comment:
I included a quick comment at https://github.com/mozilla-services/autopush/pull/798/files#diff-69cf1bc3e50483edfa74ccc88223b17dR104 where the headers are actually used. (I'll note that pjenvey notes that the string conversion safeguard wasn't really needed, I argue that it is.)

I can add a comment here, but these should probably be considered "opaque" to APNS protocols.